nomad/structs/config/tls_test.go

```diff
@@ -0,0 +1,43 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTLS_Equals_TrueWhenEmpty(t *testing.T) {
```

Comment:
In the future you should use table driven tests for this style of test: https://blog.golang.org/subtests